package.json

```diff
@@ -47,9 +47,8 @@
     "csrf": "3.0.x",
     "express": "4.15.x",
     "forever": "0.15.x",
-    "govuk_frontend_toolkit": "6.0.x",
-    "govuk_template_mustache_inheritance": "https://github.com/jonheslop/govuk_template/releases/download/v0.22.4/mustache_inheritance_govuk_template-0.22.3.tgz",
-    "hogan.js": "3.0.x",
+    "govuk_frontend_toolkit": "7.2.0",
+    "govuk_template_jinja": "^0.23.0",
```

Comment:
Need to use:
`    "govuk_template_jinja": "https://github.com/jonheslop/govuk_template/releases/download/v0.22.4/jinja_govuk_template-0.22.3.tgz",`

Reply:
Thanks, Jon

Addressed the change